src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1426,8 +1426,9 @@
                                    VerticalAlignment="Stretch"
                                    Foreground="#aaaaaa"
                                    FontWeight="Normal"
-                                   Margin="5 0 5 0">
-                            Library
+                                   Margin="5 0 5 0"
+                                   x:Name="Library">
```

Comment:
The definition is retrieved online:"Uniquely identifies XAML-defined elements in a XAML namescope. XAML namescopes and their uniqueness models can be applied to the instantiated objects, when frameworks provide APIs or implement behaviors that access the XAML-created object graph at run time."